internal/storage/db/checkOriginalURL.go

```diff
@@ -20,3 +20,15 @@ func (p *PstStorage) CheckURL(originalURL string) (string, error) {
 
 	return shortURL, errors2.ErrConflict
 }
+
+func (p *PstStorage) CheckUser(user string) error {
+
+	err := p.storage.QueryRowContext(context.Background(),
+		"SELECT user_id FROM urls WHERE user_id = $1",
+		user)
+
+	if err.Err() != nil {
+		return err.Err()
+	}
+	return nil
```

Comment:
Мне кажется не совсем верно называть переменную `err`, так как это в реальности не ошибка. Можно так же немного сократить запись.
```suggestion
	if row := p.storage.QueryRowContext(context.Background(),
		"SELECT user_id FROM urls WHERE user_id = $1",
		user); row.Err() != nil {
		return row.Err()
	}
	return nil
```

Reply:
+